jsanity-0.3.js

```diff
@@ -774,7 +636,8 @@ jSanity = {};
             'http://': 1,
             'https://': 1,
             'ftp://': 1,
-            'mailto:': 1
+            'mailto:': 1,
+            '#': 1,
```

Comment:
Maybe make this off by default, with a flag?  (similar to the yet-to-be-implemented allowRelativeURLs flag.)  I'd argue that it's not 100% guaranteed safe to allow sanitized markup to poke at various entry points on the current page without knowing the full URL.

Reply:
Thanks, will revert this change. And after investigation, IRI sanitize is non-trivial to implement, I'll remove related stuff and revert related change for this pull.